src/components/Image/Image.tsx

```diff
@@ -1,49 +1,73 @@
-import { PropsWithChildren, useRef, useState } from "react";
+import React, {
+  PropsWithChildren,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from "react";
 import ContentLoader from "components/ContentLoader";
 import ExtLink from "components/ExtLink";
 import ImagePlaceholder from "./ImagePlaceholder";
 
-export type ImageProps = {
-  src?: string;
+export type ImageProps = React.ImgHTMLAttributes<HTMLImageElement> & {
+  fallbackSrc?: string;
   isSrcLoading?: boolean;
 } & ({ href: string; title: string } | { href?: never; title?: never });
 
-type Props = {
-  img?: ImageProps;
-  className?: string;
-};
+const Image = React.forwardRef<HTMLImageElement, ImageProps>(
+  ({ className, alt = "", ...props }, forwardRef) => {
+    const ref = useRef<HTMLImageElement>(null);
+    const [isLoading, setLoading] = useState(!!props.src || props.isSrcLoading);
+    const [isError, setError] = useState(false);
 
-export default function Image({ img, className }: Props) {
-  const ref = useRef<HTMLImageElement>(null);
-  const [isLoading, setLoading] = useState(!!img?.src || img?.isSrcLoading);
+    useImperativeHandle<HTMLImageElement | null, HTMLImageElement | null>(
```

Comment:
One of the rarely used hooks in React, but very useful in this case - `ImgCropper` needs to [forward a ref to the nested `img` component inside `Image`](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/img-to-image/src/components/ImgEditor/ImgCropper.tsx#L60), but `Image` itself needs to have a `ref` to that same component. The solution is to tell any parent component to use this internal `ref` for any and all purposes, which is accomplished with this hook.